views/partials/archive-filters.blade.php

```diff
@@ -34,22 +34,22 @@
     <div class="grid">
       @if (in_array('text_search', $enabledHeaderFilters))
         <div class="grid-sm-12 grid-md-auto">
-          <label for="filter-keyword" class="text-sm sr-only"><strong><?php _e('Search', 'municipio'); ?>:</strong></label>
+          <label for="filter-keyword" class="text-sm sr-only"><strong>@php _e('Search', 'municipio'); @endphp:</strong></label>
```

Comment:
This might make more sense:

```
{{ _('Search', 'municipio') }}

Reply:
Agreed! Better to use mustache-tags where we can.